portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -65,6 +67,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
           Comment commentEntry = new Comment(keyString, comment, timestamp, nickname);
           comments.add(commentEntry);
       }
+
+      // The default value for maxComments is 10
+      long maxComments = 10;
+      String userEmail = userService.getCurrentUser().getEmail();
+      Query maxCommentQuery = new Query("MaxComment").setFilter(new FilterPredicate("email", FilterOperator.EQUAL, userEmail));
+      results = datastore.prepare(maxCommentQuery);
+      Entity entity = results.asSingleEntity();
+      // If the user has stored his preferred maxComments value before, replace the default value with stored value
```

Comment:
Better to use non-gendered language where we can, so I suggest saying "..user has stored their preferred..." rather than using "his".